Nakama/Source/NakamaCore/Public/nakama-cpp/realtime/wslay/WslayIOInterface.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "nakama-cpp/URLParts.h"
+
+namespace Nakama {
+
+// Result of async operation
+enum class NetIOAsyncResult {
+    ERR = -1,  // error, no further progress possible
+    AGAIN = 0,   // to be called again
+    DONE = 1,    // successful completion
+};
+
+class WslayIOInterface {
```

Comment:
wslay in public headers?

Reply:
yep, proprietary platforms need it and it's not behind a factory method yet